packages/amazonq/test/e2e_new/amazonq/tests/inline.test.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import '../utils/setup'
+import { Workbench, EditorView, InputBox, TextEditor, WebviewView, Key } from 'vscode-extension-tester'
+import { testContext } from '../utils/testContext'
+import { sleep, expect, pressKey, createNewTextFile } from '../utils/generalUtils'
+
+describe('Amazon Q Inline Completion / Chat Functionality', function () {
+    // this timeout is the general timeout for the entire test suite
+    this.timeout(150000)
+    let workbench: Workbench
+    let editorView: EditorView
+    let textEditor: TextEditor
+    let webviewView: WebviewView
+
+    before(async function () {
+        webviewView = testContext.webviewView
+        await webviewView.switchBack()
+        workbench = testContext.workbench
+
+        editorView = new EditorView()
+        testContext.editorView = editorView
+
+        textEditor = await createNewTextFile(workbench, editorView)
+    })
+
+    it('Inline Test', async () => {
+        await textEditor.typeTextAt(1, 1, 'Select Me')
```

Comment:
not needed here, but eventually it may make sense to abstract writing text to the editor. Allowing test writers to write text at arbitrary positions leaves a lot of room for error.